src/preprocess.py

```diff
@@ -7,7 +7,7 @@
 parser = argparse.ArgumentParser()
 parser.add_argument("--model_name", type=str, help="name of the model")
 parser.add_argument("--dataset", type=str, help="path to dataset")
-parser.add_argument("--n_spkrs", type=int, default=2, help="size of the batches")
+parser.add_argument("--n_spkrs", type=int, default=2, help="number of speakers for conversion")
```

Comment:
The current version of the code will fail if n_spkrs is not 2. We need to consider having it to be constant or something until the code support more than 2 speakers.